python/lsst/ts/mtdomecom/mock_controller.py

```diff
@@ -53,10 +53,14 @@
 )
 
 
+# Commands for the rotating part of the MTDome.
 ROTATING_COMMANDS = (
     CSCS_COMMANDS + EL_COMMANDS + LOUVERS_COMMANDS + RAD_COMMANDS + SHUTTER_COMMANDS
 )
 
+# Wait time [sec] before sending a reply. This mocks a network timeout.
+REPLY_WAIT_TIME = 600
```

Comment:
This mocking time might be too long in the test with the CSC and GUI. Maybe 10 seconds would be enough (to save the testing time)?

Reply:
The idea is that waiting for the reply times out LONG before these 600 seconds have passed. And it will only be used if `self.timeout_error` is True and by default that is False. So I will leave it as is.